src/utils.ts

```diff
@@ -77,7 +78,9 @@ async function determineNewFileName(filePath: filePath, content: fileContent , a
     }
     if (fileExtension === '.ts' || fileExtension === '.js' || fileExtension === '.jsx' || fileExtension === '.tsx') {
         return await processJavaScriptFile(filePath, content , args);
-    } else {
+    } else if(fileExtension === '.yml' || fileExtension === '.yaml') {
```

Comment:
We need to take care of the the part when the file extension can be in upper case and we are only comparing it with the lower cases. Maybe we can do that now or in future ticket

Reply:
Sure I will have this sorted out in a separate ticket